pkgs/by-name/ct/ctune/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  openssl,
+  curl,
+  ffmpeg,
+  vlc,
+  SDL2,
+  lame,
+  json_c,
+  cmake,
+  pkg-config,
+  ncurses,
+  libuuid,
+  makeWrapper,
+  pandoc,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ctune";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "An7ar35";
+    repo = "ctune";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-fy7Wb4R6HDa+cLxI5YWq67gxw1XS/iDDN4M3Rn7GYeI=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    makeWrapper
+    pandoc
+  ];
+
+  buildInputs = [
+    openssl
+    curl
+    ffmpeg
+    vlc
+    SDL2
+    lame
+    json_c
+    ncurses
+    libuuid
+  ];
+
+  cmakeFlags = [ "-DCMAKE_INSTALL_PREFIX=''" ];
+
+  patches = [ ./cmake_disable_git_clone.patch ];
+
+  meta = {
+    description = "A nice terminal nCurses (tui) internet radio player for Linux, browse and search from api.radio-browser.info";
+    homepage = " https://github.com/An7ar35/ctune";
+    changelog = "https://github.com/An7ar35/ctune/blob/master/CHANGELOG.md";
+    license = lib.licenses.agpl3Plus;
+    maintainers = with lib.maintainers; [ theobori ];
+    mainProgram = "ctune";
+    platforms = lib.platforms.linux;
```

Comment:
Is this truly an upstream constraint of this package? It sounds to me like it could work on pretty much anything that looks like unix such as Darwin or \*BSD.

Reply:
The author explicitly states that this is an application for Linux (https://github.com/An7ar35/ctune/tree/master?tab=readme-ov-file#platform).

Should I change it with `lib.platforms.unix` anyway ?

Reply:
Ah if they only intend for it to work on Linux, we can leave it at that and if someone makes it work on another platform regardless, they can just change it.